modules/secret/README.md

```diff
@@ -0,0 +1,96 @@
+# terraform-google-secret-manager
+
+This modules makes it easy to create a single Google Secret Manager secret. It supports using custom KMS keys (CMEK) for encrypting the secrets.
+It also supports secret rotation and can configure the rotation notification to be sent to given pubsub topics.
+
+## Usage
+
+Basic usage of this module is as follows:
+
+```hcl
+module "secret-manager" {
+  source  = "GoogleCloudPlatform/secret-manager/google//modules/secret"
+  version = "~> 0.3"
+
+  project_id  = var.project_id
+  name        = "secret-1"
+  secret_data = "secret information"
+}
+```
+
+Functional examples are included in the [examples](./examples/) directory.
```

Comment:
Let's call out the specific example which uses this particular submodule.

Reply:
Done in https://github.com/GoogleCloudPlatform/terraform-google-secret-manager/pull/76/commits/fc61bc59118354620bb39c52f73a4384c6dc725e